arrow/util.py

```diff
@@ -23,6 +28,42 @@ def is_timestamp(value):
         return False
 
 
+def datetime_from_timestamp(timestamp, tz=None):
```

Comment:
Maybe this function could be called `windows_datetime_from_timestamp` to make it clear what it's purpose is.

Reply:
Sure, will change that.